### PR TITLE
Don't add invalid paths to IMG_PATH

### PR DESCRIPTION
**Pull request by ericfrederich**
This happens to work today because directories are only one level deep
under images so "path" was always the same as "_root".

Also, rename variables to match os.walk's documentation.

### CODE REVIEW

**Comment by goanpeca** [comment]
Any update on this one @ccordoba12, @ericfrederich ?

File: spyderlib/baseconfig.py [hunk 1]
```diff
@@ -179,9 +179,9 @@ def add_image_path(path):
         return
     global IMG_PATH
```
Comment (Nodd) [inline spyderlib/baseconfig.py]: Nothing to do with this PR, but why is this line necessary? `IMG_PATH` will be taken from the outer namespace. It would be necessary if there was `IMG_PATH = something`, but here we're only `.append()`ing to it.

**Comment by Nodd** [comment]
We should merge this one, I see nothing wrong with this code.

**Comment by ccordoba12** [comment]
Ok, merging then per @Nodd's wishes ;-)
